students/BrandonHenson/Lesson2/series.py

```diff
@@ -0,0 +1,46 @@
+'''
+Brandon Henson
+4/2/18
+Lesson 2 series.py
+using index and math to make functions and assertions
+'''
+
+def fibonacci(n):
+    """This function prints out the digit in the Fibonacci series that is in the index number defined by the parameter."""
+    fibo_list = [0,1]
+    for i in range(2, n+1):
+        x = fibo_list[i-2] + fibo_list[i-1]
+        fibo_list.append(x)
+    #printed to test if I'm getting the index number
+    #print(fibo_list[n])
```

Comment:
Print statements can be a useful debugging tool while you're developing, but you should remove them entirely (rather than comment out) before committing. Once you've got your program working, you can use tests, such as the `assert` statements below to ensure that your code is performing as expected.